### PR TITLE
RDP Minutes to Idle Timeout - Update 4

### PR DESCRIPTION
**Pull request by countchappy**
Added an option to alert the user if they were disconnected due to inactivity. Additionally, if the user specifies a number of minutes to idle that is above the maximum, it will set the number of minutes to idle at the max (used to be that it would set the minutes to idle to zero or never idle out).
_This is continued off of PR #342 and ticket #221_

### CODE REVIEW

**Comment by sparerd** [comment]
Sorry it's taken me so long to review this. Comments incoming

**Comment by sparerd** [comment]
@countchappy sorry... forgot to submit my review. it was sitting there for 4 days until Sean said he couldn't see it either :\

**Comment by kmscode** [comment]
@sparerd and I are looking to get a 1.75 release candidate built within the next few days/week - any chance to could review the comments that were just posted and address them soon so we can accept this PR to include in the release?

Thanks again for your efforts, greatly appreciated!

File: mRemoteV1/Connection/Protocol/RDP/Connection.Protocol.RDP.cs [hunk 1]
```diff
@@ -129,11 +130,17 @@ public override bool Initialize()
                 SetCredentials();
                 SetResolution();
                 _rdpClient.FullScreenTitle = _connectionInfo.Name;
+
+                _alertOnIdleDisconnect = _connectionInfo.RDPAlertIdleTimeout;
                 
                 // Set the timeout to the default (zero) if it is out of bounds.
-                if (_connectionInfo.RDPMinutesToIdleTimeout < 0 || _connectionInfo.RDPMinutesToIdleTimeout > 240)
+                if (_connectionInfo.RDPMinutesToIdleTimeout < 0)
```
Comment (countchappy) [inline mRemoteV1/Connection/Protocol/RDP/Connection.Protocol.RDP.cs]: @sparerd Will correct and commit.

Comment (sparerd) [inline mRemoteV1/Connection/Protocol/RDP/Connection.Protocol.RDP.cs]: Thanks!

**Comment by countchappy** [comment]
@sparerd Just pushed the commit to correct the number rather than truncate it into the range.

Now trying to correct and have the alert support different languages.

**Comment by sparerd** [comment]
Looks great now! I'm happy with the PR as it is. It's up to you if you'd like to add the localizations in as well, or do that later. We will be releasing 1.75 RC once this PR is done (probably this weekend).

**Comment by countchappy** [comment]
@sparerd I do not believe I can get the localization support done by this weekend

I will commit those changes and submit a PR at a later date 😄 Thanks again for all the help!

**Comment by sparerd** [comment]
@countchappy no problem! Thanks for taking the time to help out the project!

**Pull request merged**